### DISCUSSION TITLE
Options section in hurl file

### DISCUSSION THREAD

**Discussion by cesarsaenz** [Orange-OpenSource/hurl#1703] [Q&A] [answered]
Hi trying to use other options in the hurl file I see an example of 

[Options]
location: true

which works perfectly, was trying to user other options, like resolve but getting error
example:
[Options]
resolve: example.com:443:23.50.52.35

but does not work, get 
^ the HTTP method <resolve> is not valid. Valid values are GET, HEAD, POST, PUT, DELETE, CONNECT, OPTIONS, TRACE, PATCH

**Comment by jcamiel** [comment]
Hi, effectively not every command line option can be set in [Options] section. We need to implement the relevant ones, it's just a matter of time!

You can see the options you can override per request here https://hurl.dev/docs/request.html#options I will add resolve to our TODO list.

**Comment by jcamiel** [comment]
Created issue here => #1711

**Comment by jcamiel** [comment, marked as answer]
@cesarsaenz `--resolve` option can be used now per request, it is on branch master, and should be landing in the next Hurl release.
